runtime/in-memory-storage.js

```diff
@@ -118,18 +118,18 @@ export class InMemoryCollection extends InMemoryStorageProvider {
     this.description = view.description;
   }
 
-  get(id) {
+  async get(id) {
```

Comment:
I guess doing this isn't strictly necessary, but makes it easier to avoid bugs where we assume this is always synchronous.

Reply:
Yup. I wanted to do this now so that all these changes were well separated from adding firebase into the cauldron.